### ISSUE TITLE
Échec de l'installation (debian 8)

### ISSUE THREAD

**Issue by Shnoulle** [YunoHost-Apps/gogs_ynh_old#7]
When try to install in own domain : stop at cd:

```
+ sudo cp ../conf/app.ini /opt/gogs/custom/conf
cp: impossible de créer le fichier standard « /opt/gogs/custom/conf »: Aucun fichier ou dossier de ce type
+ sudo sed -i s@yuno_repo@/home/yunohost.app/gogs@g /opt/gogs/custom/conf/app.ini
sed: impossible de lire /opt/gogs/custom/conf/app.ini: Aucun fichier ou dossier de ce type
....
```

<s>Surely need to create dir before cp. I make a request if it fix it :) </s>

No debian 8 repo :https://packager.io/gh/pkgr/gogs

```
+ sudo apt-get update
E: Le pilote pour la méthode /usr/lib/apt/methods/https n'a pu être trouvé.
+ sudo apt-get -y install gogs apt-transport-https
Lecture des listes de paquets…
Construction de l'arbre des dépendances…
Lecture des informations d'état…
E: Impossible de trouver le paquet gogs
```

**Comment by Shnoulle** [comment]
Fixed with https://github.com/mbugeia/gogs_ynh/pull/8

**Comment by balazsgrill** [comment]
I had similar error during installation.

Full log: https://gist.github.com/balazsgrill/a50a6b25f94030c58bb2

**Comment by mbugeia** [comment]
Should be fixed with #10
